db-engine/src/main/java/com/splicemachine/db/iapi/store/access/TransactionController.java

```diff
@@ -804,6 +804,52 @@ TransactionController startNestedUserTransaction(
 			boolean flush_log_on_xact_end)
         throws StandardException;
 
+    /**
+     * Get a nested internal transaction.
+     * <p>
+     * A nested internal transaction is used in place of a user transaction
+	 * when a DML statement is executing its operation tree and a new writable
+	 * child transaction needs to be created with the possibility that within
```

Comment:
Indentation is weird in this newly added block. Comment lines start to shift to the right and method parameters are not aligned.

Reply:
Thanks, I've fixed the indentation in the whole file.